pkgs/by-name/ro/roslyn-ls/package.nix

```diff
@@ -6,64 +6,43 @@
   stdenvNoCC,
   testers,
   jq,
-  writeText,
   runCommand,
   expect,
 }:
 let
   pname = "roslyn-ls";
+  # add all supported sdks to minimize deps.json
+  # see https://github.com/dotnet/roslyn/blob/main/eng/targets/TargetFrameworks.props
   dotnet-sdk =
     with dotnetCorePackages;
-    sdk_10_0
-    // {
-      inherit
-        (combinePackages [
-          sdk_9_0
-          sdk_8_0
-        ])
-        packages
-        targetPackages
-        ;
-    };
-  # need sdk on runtime as well
-  dotnet-runtime = dotnetCorePackages.sdk_10_0;
+    (combinePackages [
+      sdk_10_0
+      sdk_9_0
```

Comment:
Does it definitely need all of these SDKs?  Previously it was using only sdk 10 and the packages from 8+9.